pipelines/integration_resolver_pipeline_environment_pass.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: integration-resolver-pipeline-pass
```

Comment:
Let's name this something in line with it's use-case, like `integration-environment-pipeline` or similar.

Reply:
that sounds much better,  I picked the below if no objection 

`  name: integration-environment-pipeline-pass`